src/components/FieldGuide.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StepThrough } from 'zooniverse-react-components';
+import { Markdown } from "markdownz";
+import { getSubjectLocation } from '../lib/get-subject-location';
+
+class FieldGuide extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      activeCard: null,
+      groupedItems: []
+    }
+
+    this.renderItem = this.renderItem.bind(this);
+    this.groupItems = this.groupItems.bind(this);
+    this.renderActiveCard = this.renderActiveCard.bind(this);
+    this.deactivateCard = this.deactivateCard.bind(this);
+  }
+
+  componentWillMount() {
+    this.groupItems();
+  }
+
+  groupItems() {
+    let groupedItems = [];
+
+    if (this.props.guide && this.props.guide.items) {
+      let size = 10;
+      let items = this.props.guide.items.slice();
+
+      while (items.length > 0)
+        groupedItems.push(items.splice(0, size));
+    }
+    this.setState({ groupedItems })
+  }
+
+  activateCard(activeCard) {
+    this.setState({ activeCard });
+  }
+
+  deactivateCard() {
+    this.setState({ activeCard: null });
+  }
+
+  renderItem(items, index) {
+    return (
+      <div className="field-guide" key={`FIELD_GUIDE_PAGE_${index}`}>
+        {items.map((item, i) => {
+          const resource = item.icon;
+          const src = this.props.icons[resource].src;
```

Comment:
IE11 spits out an error here - "Unable to get property 'src' of undefined or null reference".

Nyeeerp? Sounds like IE11 has an issue with the... resource... key? Whatever, easiest fix here is to apply a quick existential check before assigning the value of `src`

Reply:
The error happens in Edge too, so an existential check seems required for the Field Guide to work.